code/modules/mod/mod_activation.dm

```diff
@@ -191,6 +191,9 @@
 	SEND_SIGNAL(src, COMSIG_MOD_TOGGLED, user)
 	return TRUE
 
+/obj/item/mod/control/proc/is_sealed(obj/item/clothing/part)
+	return part.icon_state == "[skin]-[part.base_icon_state]-sealed"
```

Comment:
This is pretty scuffed, please consider adding a variable or some other kind of control flag that keeps track of whether a part was sealed